tedana/tests/test_tedana.py

```diff
@@ -47,11 +47,11 @@ def test_outputs():
      'T1gs.nii',
      'tsoc_orig.nii',
      'tsoc_nogs.nii',
-     '.cc_temp.nii.gz',
-     '.fcl_in.nii.gz',
-     '.fcl_out.nii.gz',
-     '__clin.nii.gz',
-     '__clout.nii.gz',
+     # '.cc_temp.nii.gz',
```

Comment:
Can we just delete these, rather than commenting them out ?

Reply:
Definitely! I suppose I should have done that in the first place, but I wasn't confident this would work in a single shot. I'll remove them now.